build.gradle

```diff
@@ -3,13 +3,13 @@ plugins {
   id 'checkstyle'
   id 'idea'
   id 'jacoco'
+  id 'se.patrikerdes.use-latest-versions' version '0.2.18'
```

Comment:
Is this needed?

Reply:
This will be removed from each repo before release but is being used to keep track of patch version upgrades for the final version bumps